examples/resources/unleash_role/resource.tf

```diff
@@ -1,3 +1,18 @@
+import {
+  id = 5
+  to = unleash_role.project_member_role
+}
+
+resource "unleash_role" "project_member_role" {
+  permissions = [{
+    name = "CREATE_PROJECT"
+    }, {
+    name = "UPDATE_PROJECT"
+    }, {
+    name = "DELETE_PROJECT"
+  }]
+}
+
```

Comment:
Intention here is to import the project member role and override the permissions